.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+# Further reading: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+# and also: https://github.com/pypa/gh-action-pypi-publish
+
+name: Upload Python Package to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
Huh neat, didn't know you could do `3.x` to pick the highest version.

Reply:
I can put `3.8` here, I guess, though I doubt we use anything specific from `3.7` or `3.8`, don't we? All in all, it should work on `3.6` regardless of that.